android/app/build.gradle

```diff
@@ -231,7 +234,7 @@ dependencies {
         implementation jscFlavor
     }
 
-    implementation platform("com.google.firebase:firebase-bom:20.0.2")
+    implementation platform("com.google.firebase:firebase-bom:29.0.3")
```

Comment:
Just curious why we need to update this?

Reply:
Firebase perf was also blocking release builds. I don't recall the exact error message, but after resolving onfido this one needed to be fixed too.

Reply:
I was curious, so here's the exception: adding other repositories does not fix the issue, so I believe it was also due to BinTray being deprecated -- as our previous Firebase version was very old.

<img width="1144" alt="Screenshot 2021-12-21 at 14 22 40" src="https://user-images.githubusercontent.com/10736861/146945616-6de473d1-1450-44c2-aaee-ecebf966fe34.png">

Reply:
I mentioned this in a commit message, but probably should have added a note to the PR description too!

Reply:
Ok cool. Just double-checking because I have a feeling the various firebase libs need to be updated at the same time. This is not really a blocker if everything is working as expected.

Reply:
https://github.com/Expensify/Expensify/issues/192204